about.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Lato:wght@300;900&family=Poppins:wght@200&family=Roboto:ital,wght@0,400;1,100&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/about.css" />
+    <title>Tech Africa</title>
+  </head>
+
+  <body>
+    <main>
+      <header>
+        <div class="top-banner">
+          <ul>
+            <li>
+              <a href="#">
+                <img src="/images/fa-icon.png" alt="" />
+              </a>
+            </li>
+
+            <li>
+              <a href="#">
+                <img src="/images/twitter-link.png" alt="" />
+              </a>
+            </li>
+            <li><a href="#">English</a></li>
+            <li>
+              <a href="#">My page</a>
+            </li>
+            <li><a href="#">Logout</a></li>
+          </ul>
```

Comment:
- [ ] Please ensure that the spacing between links and also the padding around the navbar matches that of the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/163595512-f7bf01fe-1158-4878-9fa1-0d655e1f7be6.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/163595619-22e60e20-bcb7-4efa-b1af-34a0b43a1483.png)